### ISSUE TITLE
Save/Store a build query

### ISSUE THREAD

**Issue by ghost** [bcgov/lucy-web#292]
As a Data Entry+ user I want to be able to save and store selected queries to my profile page so that I can reuse commonly used queries and share them with other users.

**Additional Context**
- dependent on issue #290 (Implement a query)
- saved query would be assigned to user profile
- assign date/some identifier to each query
- Add a short comment (50 characters to help identify the query)
- user should be able to view the query results to confirm that query has been built correctly before saving the query
- the ability to delete a saved query should exist
- the ability to edit a query is out of scope for mvp

**Acceptance Criteria**
- [ ] Given that I am creating a query, when I press the "save query" button, ...
